runtime/Stdlib.res

```diff
@@ -101,6 +101,23 @@ external import: 'a => promise<'a> = "%import"
 
 let panic = Error.panic
 
+/**
+Throw and exception which will stop execution.
+
+## Examples
+
+```rescript
+let error = Error.make("Everything is upside down.")
+
+if 5 > 10 {
+  error->throw
+} else {
+  Console.log("Phew, sanity still rules.")
+}
+```
+*/
+external throw: Error.t => 'a = "%raise"
```

Comment:
Should I add the external here or just leave it in the `Stdlib_Error.res` module?

Reply:
Done